test/JLM_Tests/src/org/openj9/test/java/lang/management/RemoteProcess.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+package org.openj9.test.java.lang.management;
+import java.io.File;
+import java.io.UnsupportedEncodingException;
+
+public class RemoteProcess {
+
+	/**
+	 * Start the RemoteProcess with jmx enabled, for example:
+	 * -Dcom.sun.management.jmxremote.port=9999
+	 * -Dcom.sun.management.jmxremote.authenticate=false
+	 * -Dcom.sun.management.jmxremote.ssl=false
+	 */
+	public static void main(String[] args) {
+		System.out.println("=========RemoteProcess Starts!=========");
+		for (int i = 0; i < 60; i++) {
+			try {
+				int[] a = new int[1024 * 1024];
+				new String("hello").getBytes("Unsupported");
+			} catch (UnsupportedEncodingException e) {
+				/* Expected exception from getBytes() hence ignored */
+			} catch (Exception e) {
+				e.printStackTrace();
```

Comment:
What exception is anticipated here? Why should the test not fail if it is caught?

Reply:
I have dynamically configured the dump options to the below event in the remote test

diagBeanRemote.setDumpOptions("java:events=catch,filter=java/io/UnsupportedEncodingException,range=1..1,file=" + dumpFilePath);

and included the below line in the RemoteProcess to get UnsupportedEncodingException:
new String("hello").getBytes("Unsupported");

Expected Exception is UnsupportedEncodingException thrown by String.getBytes()

Reply:
Yes, UnsupportedEncodingException is caught on line 41; I was asking what is caught (that should be ignored) on line 43.

Reply:
@chandrams Can you answer my question about that is caught on line 43?

Reply:
Sorry, missed replying to this one earlier.  Will remove that catch as I don't see any other exception expected there.